src/platform/webengine/BlinkWebViewProfileHelper.cpp

```diff
@@ -17,7 +17,8 @@
 
 #include "BlinkWebViewProfileHelper.h"
 #include "webos/webview_profile.h"
-#include <string.h>
+
+#include <cstring>
```

Comment:
Why cstring here?

Reply:
It is used for string comparison using  strcmp